docs/index.md

```diff
@@ -1,9 +1,46 @@
 
 # Welcome to WheelNext
 
-## Introduction
-
-Content to be added here.
+## What is WheelNext and Why should you participate !?
+
+WheelNext is an open-source initiative (<https://github.com/wheelnext> & <https://wheelnext.dev/>) aiming to improve the
+user-experience in the Python packaging ecosystem, specifically around the scientific computing and machine/deep learning space.
+The current state of Python packaging was designed and implemented at a time before the increasingly complex needs these use-cases demand were prevalent.
+Through WheelNext, we aim to evolve Python packaging standards to better address the needs of this important segement, while retaining its effectiveness for the overall ecosystem.
+
+Many of the problems and gaps are summarized on the following website: <https://pypackaging-native.github.io/>
+Objective: To improve the state of packaging for Python native code, making it easier to build, distribute, and install
+Python packages with complex native dependencies.
+
+### Key Issues Identified:
```

Comment:
It looks to me like @warsaw's comments on some of the bullets are on point, and can be made about the other bullet points as well. E.g., we're probably going to introduce more rather than less Complexity.

It's probably best to be more concrete. The key issues are lack of support for:
- GPUs and other accelerators
- CPU capabilities beyond the CPU family (SIMD instruction sets)
- dependencies meant to be used as a single runtime or shared library rather than vendored (e.g., OpenMP, MPI, BLAS, LAPACK)
- paper cuts specific to packaging complex native libraries as wheels (e.g., symlink support)

Reply:
@rgommers do you mind making suggestions to the PR ? Much easier to capture a phrasing that fairly represent your thinking if it comes directly from you

Reply:
Sure, if you're happy for me to rewrite this whole section. I'll try now, and point you at a commit since inline suggestions won't work for such a large block.

Reply:
Here it is: https://github.com/rgommers/wheelnext/commit/ebc4e6faf39a365b385670a2de1ccdf6c83b2740

It could be improved further, but I think it's already in reasonable shape. A key point is to remove the Complexity, Tooling Fragmentation, Lack of Standardization, Dependency Management items, because some of those can be argued with and none of those are things WheelNext is going to solve.

Reply:
Perfect. Thanks it was merged